### ISSUE TITLE
RAM and VRAM Issues for Cargo Plane

### ISSUE THREAD

**Issue by Flamity** [przemykomo/simple-planes#38]
While a Cargo Plane is on screen/loaded it will constantly use TONS of RAM and VRAM. This is only for the Cargo Plane and we're not too sure why. Me and my friends suspect that it's saving logging information but I honestly have no clue. The mod would cause our games to get up to MAX 16gb of RAM and for my friend, 20gb. It would also clutter the VRAM making the game unplayable to the point where it would crash. Please fix.

**Comment by adoxentor** [comment]
mod, minecraft, loader version? and full mod list if you can.

**Comment by Flamity** [comment]
I did some testing just now with the simple planes mod. I loaded the mod without anything else in the mods folder. I'm using mod version 1.12.2-3.0.1.7 on forge loader version 1.12.2-14.23.5.2854. I loaded a new single player world and as soon as I placed the cargo plane my RAM would go up about 10-12MB per second until the game crashed, and my VRAM the same. As soon as I destroy the cargo plane it keeps all of it in memory too. So I don't think the other mods had anything to do with it. Hope this helps, I can give the mod list anyways if needed thanks.
